monitoring/tools/check_dead_queues.py

```diff
@@ -15,18 +15,22 @@ def check_deadletter_queues(
 ):
     bad_queue_messages = []
     dead_queues = get_queues(contains="deadletter")
+    environment = None
     for dead_queue in dead_queues:
         queue_size = int(dead_queue.attributes.get("ApproximateNumberOfMessages", 0))
         if queue_size > 0:
             queue_name = dead_queue.url.split("/")[-1]
-            bad_queue_messages.append(f"Queue {queue_name} has {queue_size} items")
-
-    bad_queues_str = "\n".join(f" * {q}" for q in bad_queue_messages)
-    message = dedent(
-        f"Found {len(bad_queue_messages)} dead queues with messages:\n{bad_queues_str}"
-    )
+            if environment is None:
+                environment = queue_name.split("-")[1].upper()
```

Comment:
If there's a manual queue, with a name that doesn't include `-`, this will fail. Can we wrap it in `try/except` please.